src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -364,3 +376,63 @@
     <br />
     <br />
 </div>
+
+<hr />
+<div class="nav-container">
+    <h1>Bit Navs</h1>
+    <h2>Basic nav with sample links</h2>
+    <BitNav Style=" width: 208px;
+                    height: 350px;
+                    box-sizing: border-box;
+                    border: 1px solid #eee;
+                    overflow-y: auto;"
+            Class="bit-nav-simple-link"
+            NavLinks="BasicNavLinks"
+            AriaLabel="Nav basic example"
+            SelectedKey="key3">
+    </BitNav>
+
+    <br />
+    <h2>Nav with wrapped link text and no tooltips</h2>
+    <BitNav Style=" width: 208px;
+                    height: 350px;
+                    box-sizing: border-box;
+                    border: 1px solid #eee;
+                    overflow-y: auto;"
+            Class="bit-nav-wrapped-link"
+            SelectedKey="key5"
+            AriaLabel="Nav example with wrapped link text"
+            NavLinks="BasicNoToolTipNavLinks">
+    </BitNav>
+
+    <br />
+    <h2>Nav similar to the one in this demo app</h2>
+    <BitNav Style=" width: 300px;"
+            NavLinks="BasicNoUrlNavLinks"
+            AriaLabel="Nav example similar to one found in this demo page">
+    </BitNav>
+
+    <br />
+    <h2>Nav with nested links</h2>
+    <BitNav 
```

Comment:
what is the new-line character at the end of this line?
it's a code formatting issue.